JS/Lesson16/savas/script.js

```diff
@@ -1,75 +1,173 @@
+console.log('- - - - - - - - - - - - - - - - - EXERCISE 1 - - - - - - - - - - - - - - - - - ');
 // 1. Write a loop that prints out the numbers from 1 to 10.
 for (let i = 1; i<=10; i++) {
 	console.log(i);
 }
-console.log('- - - - - - - - - - - - - - - - - ');
+
+console.log('- - - - - - - - - - - - - - - - - EXERCISE 2 - - - - - - - - - - - - - - - - - ');
 // 2. Write a loop that prints out the odd numbers from 1 to 20.
 // 1,3,5,7...
 for (let i = 1; i<20; i = i+2){
 	console.log(i);
 }
 
-console.log('- - - - - - - - - - - - - - - - - ');
+console.log('- - - - - - - - - - - - - - - - - EXERCISE 3 - - - - - - - - - - - - - - - - - ');
 // 3. Write a loop that prints out the even numbers from 1 to 20.
 // 2, 4, 6...
 for (let i =2; i<=20; i+=2) {
 	console.log(i);
 }
 
-console.log('- - - - - - - - - - - - - - - - - ');
+console.log('- - - - - - - - - - - - - - - - - EXERCISE 4 - - - - - - - - - - - - - - - - - ');
 // 4. Write a loop that finds the maximum number in an array of numbers.
 const findMaxNumberInArray = [300, 10, 23, 42, 505, 12, 7, 4, 900];
+let num = findMaxNumberInArray[0];
+for (let i = 0; i < findMaxNumberInArray.length; i++) {
+	if (num < findMaxNumberInArray[i+1]) {
```

Comment:
When the loop will come to the last element of the array, expression `findMaxNumberInArray[i+1]` will try to get an element outside array (non-existent). It does not cause a problem here but may do that in a different situation.
A safer approach would be to set the initial value of i to 1 (`for (let i = 1; i < findMaxNumberInArray.length; i++) {`) as you covered the element with index 0 already. Then, you'd need to check current index instead: `if (num < findMaxNumberInArray[i]) {`